vendor/github.com/containers/image/internal/testing/explicitfilepath-tmpdir/tmpdir.go

```diff
@@ -0,0 +1,29 @@
+// Package tmpdir is a TESTING-ONLY utility.
```

Comment:
(Completely separately from the substantial discussion of the fix in https://github.com/containers/image/pull/646 ), did `vndr` change somehow to add these test-only sub packages?

Reply:
What I see is that a local `vndr` run adds those, but the one in Travis does not. Need to investigate more.

Reply:
Opened https://github.com/containers/skopeo/pull/681 to migrate to go modules. I think it's a good time for that move (instead of investing time into debugging `vndr` issues).